src/Assembler.hpp

```diff
@@ -126,6 +126,7 @@ class shasta::AssemblerInfo {
     // Performance statistics.
     double assemblyElapsedTimeSeconds = 0.;
     double averageCpuUtilization;
+    string peakMemoryUsage = "";
```

Comment:
Perhaps store it as a uint64_t, in bytes? That would make it easier to do computations with it if we want to - for example report the number of bytes used per input base.